apps/widgets/app/[businessId]/cart/page.tsx

```
@@ -0,0 +1,63 @@
+    };
+
+    return (
+      <>
+        <div
+          className="inline-flex bg-transparent p-0 m-0"
+          style={{ color: colors.text[900], width: 'fit-content', margin: 0, padding: 0, overflow: 'visible' }}
+        >
+          <CartWidget businessId={business.id} colors={colors} themeName={themeName} widgetId={widgetId} />
+        </div>
+        <style
+          dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)